setup.py

```diff
@@ -195,6 +195,7 @@ def cython_extensions_and_necessity():
   extensions = [
       _extension.Extension(
           name=module_name,
+          language='c++',
```

Comment:
@kpayson64 I'm slightly worried this would actually ask Python to link using g++ instead of gcc, which would drag in libstdc++ automatically, and would thus cause breakage of pre-compiled binaries being shipped on other distributions.

Can we run a sanity check on that using the distribution build test ? We should still be able to load the resulting python extension on older distributions that is. Or more directly, the resulting .so file (or whatever name that has) shouldn't have any reference to libstdc++.

You can tell using `ldd`. For example:

```
$ ldd protoc
        linux-vdso.so.1 (0x00007fff2a60e000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f858ac76000)
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f858aa5c000)
        libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f858a6da000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f858a3d6000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f858a1bf000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f8589e1f000)
        /lib64/ld-linux-x86-64.so.2 (0x000055fad2c0b000)
```

Here we can clearly see the libstdc++.so.6 dependency.